test/src/e2e_vm_tests/test_programs/should_fail/attributes_invalid_abi_names/stdout.snap

```diff
@@ -0,0 +1,63 @@
+---
+source: test/src/snapshot/mod.rs
+---
+> forc build --path test/src/e2e_vm_tests/test_programs/should_fail/attributes_invalid_abi_names
+exit status: 1
+output:
+    Building test/src/e2e_vm_tests/test_programs/should_fail/attributes_invalid_abi_names
+   Compiling library std (sway-lib-std)
+   Compiling contract attributes_invalid_abi_names (test/src/e2e_vm_tests/test_programs/should_fail/attributes_invalid_abi_names)
+error
+ --> test/src/e2e_vm_tests/test_programs/should_fail/attributes_invalid_abi_names/src/lib.sw:6:1
+  |
+4 | struct MyStruct {}
+5 | 
+6 | #[abi_name(name = "SameName")]
+  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Duplicated name found for renamed ABI type.
```

Comment:
We can heavily benefit here from expressive diagnostics, by pointing to the source of the other name. Because this name can be a name of the type which is not in the same file, or even of an another one `#[abi_name]` that is also not in the same file.

The proposal is:
- to use expressive diagnostics to point to the other name.
- to extend the test and cover the cases where the duplicated name is a name of another type, or another `#[abi_name]` in the current or some other file.

Another suggestion is to underline only the name itself, "SameName" in this example, because that's where the issue actually is. The rest of the attribute usage is perfectly fine and underlining it gives a misleading impression that the usage of the attribute is wrong.